assets/js/modules/adsense/settings/adsense-settings.js

```diff
@@ -179,11 +210,47 @@ class AdSenseSettings extends Component {
 							}
 						</Fragment>
 						: <Fragment>
-							{	__( 'The AdSense code has', 'google-site-kit' ) } {
-								useSnippet
-									? __( 'been placed on your site.', 'google-site-kit' )
-									: __( 'not been placed on your site.', 'google-site-kit' )
-							}
+							<div className="googlesitekit-settings-module__meta-items">
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'Publisher ID', 'google-site-kit' ) }
+									</p>
+									<h5 className="googlesitekit-settings-module__meta-item-data">
+										{ accountID }
+									</h5>
+								</div>
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'AdSense Code', 'google-site-kit' ) }
+									</p>
+									<h5 className="googlesitekit-settings-module__meta-item-data">
+										{ useSnippet && __( 'The AdSense code has been placed on your site.', 'google-site-kit' ) }
+										{ ! useSnippet && __( 'The AdSense code has not been placed on your site.', 'google-site-kit' ) }
+									</h5>
+								</div>
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'Site Status', 'google-site-kit' ) }
+									</p>
+									<h5
+										className="googlesitekit-settings-module__meta-item-data"
+										dangerouslySetInnerHTML={
+											sanitizeHTML( linkToAdSenseSites, {
+												ALLOWED_TAGS: [ 'a' ],
+												ALLOWED_ATTR: [ 'href' ],
+											} )
+										}>
+									</h5>
+								</div>
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'Account Status', 'google-site-kit' ) }
+									</p>
+									<h5 className="googlesitekit-settings-module__meta-item-data">
+										{ this.generateAccountStatusLabel() }
+									</h5>
+								</div>
```

Comment:
Can you arrange these fields a bit differently? This was not originally described, but the current order is a bit confusing and it's also rather cluttered being all in one row. Let's change it as follows:

* Row 1: Publisher ID, then Account Status, then Site Status
* Row 2: AdSense Code

This is more in line with how Analytics does it too, general account information first, then an extra row regarding the snippet, so you should be able to just add an extra `.googlesitekit-settings-module__meta-items` wrapper.